pkg/plugin/plugin.go

```diff
@@ -34,12 +38,12 @@ type Information struct {
 
 type Plugin struct {
 	command      *exec.Cmd
-	stdOut       io.ReadCloser
-	stdErr       io.ReadCloser
 	mutex        sync.RWMutex
 	status       Status
 	info         *Information
 	reverseProxy *httputil.ReverseProxy
+	BinPath      string
+	ID           int64
```

Comment:
i don't really see the issue. It could be covenient for a plugin to knwo its ID without asking the manager. But yes it is not mandatory to have it here.
Do you want me to change this?